libcst/tests/pyre/simple_class.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+# fmt: off
+from typing import Sequence
+
+
+class Item:
+    def __init__(self, n: int) -> None:
+        self.number = n
+
+
+class ItemCollector:
+    def get_items(self, n: int) -> Sequence[Item]:
+        return [Item(i) for i in range(n)]
+
+
+collector = ItemCollector()
+items = collector.get_items(3)
```

Comment:
The type of `collector.get_items` (a callable) is missing.

Reply:
Hm, when I tested this I was able to get the type of get_items as part of the JSON output:
{"location":{"start":{"line":52,"column":8},"stop":{"line":52,"column":27}},"annotation":"typing.Callable(test_prop.ItemCollector.get_items)[[Named(n, int)], typing.Sequence[test_prop.Item]]"},

my test file looks like this:

```
import typing
from typing import (
    TYPE_CHECKING,
    Any,
    AsyncGenerator,
    AsyncIterable,
    AsyncIterator,
    Awaitable,
    BinaryIO,
    Callable,
    ClassVar,
    Collection,
    DefaultDict,
    Dict,
    Final,
    Generator,
    Generic,
    Iterable,
    Iterator,
    List,
    Mapping,
    NamedTuple,
    NewType,
    Optional,
    Protocol,
    Sequence,
    Set,
    SupportsInt,
    Tuple,
    Type,
    TypeAlias,
    TypeVar,
    Union,
    cast,
    overload,
)
from mypy_extensions import TypedDict
from typing_extensions import Literal

# pyre-strict

class Item:
    def __init__(self, n: int) -> None:
        self.number = n

class ItemCollector:
    def get_items(self, n: int) -> Sequence[Item]:
        return [Item(i) for i in range(n)]

collector = ItemCollector()
reveal_type(collector.get_items)
items = collector.get_items(3)

```

Let me know if I'm reproing something differently, happy to take another look!

Reply:
Correct, my comment was outdated, the type shows up after change to use 0.0.38.